model/src/main/java/org/cbioportal/model/CopyNumberSeg.java

```diff
@@ -1,17 +1,20 @@
 package org.cbioportal.model;
 
 import java.io.Serializable;
+import java.math.BigDecimal;
 
-public class CopyNumberSegment implements Serializable {
+public class CopyNumberSeg implements Serializable {
```

Comment:
As I said in a previous PR, I'm trying to match model names to database table names just in case we use an ORM library in the future.